packages/fxa-auth-server/lib/senders/emails/partials/changePassword/index.txt

```diff
@@ -0,0 +1,2 @@
+change-password-plaintext = "If you suspect that someone is trying to gain access to your account, please change your password. { $passwordChangeLink }"
```

Comment:
I think, since you’re rendering `passwordChangeLink` in EJS format, the inline `{ $passwordChangeLink }` bit can be removed

Reply:
Ah oops, yeah I missed that. Fixed.